hw/ip/flash_ctrl/dv/cov/flash_ctrl_cov.vRefine

```diff
@@ -0,0 +1,59 @@
+    </ccf-files>
+  </information>
+  <rules>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/tck_i" entityType="toggle" excTime="1652803318" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/tdi_i" entityType="toggle" excTime="1652803318" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/tms_i" entityType="toggle" excTime="1652803318" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/tdo_o" entityType="toggle" excTime="1652803318" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/bist_enable_i" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/scanmode_i" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/scan_en_i" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/scan_rst_ni" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/flash_power_ready_h_i" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
```

Comment:
these special flash power / test / voltage signals are like the flash jtag signals, we should have a connectivity test. Is that covered separately already?

Reply:
Same comment as above.